src/components/PageList.vue

```diff
@@ -245,6 +273,12 @@ export default {
 		},
 	},
 
+	watch: {
+		filterString() {
+			this.getContentFilteredPagesDebounced.apply(this)
```

Comment:
What's the reason for not just calling `this.getContentFilteredPageDebounced()` without the `.apply(this)`? Please forgive my question, it's just that I know the former and never saw the `.apply(this)` style so far.